timeline-tests/test/Spec/Data/Timeline.hs

```diff
@@ -0,0 +1,250 @@
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Spec.Data.Timeline where
```

Comment:
We should stop doing this a different way in every package (I'm guilty too!):

* `aws-arn`: `module Foo.Bar.Baz.Test`
* `wai-handler-hal`: `module Foo.Bar.BazTest`
* `timeline-tests`: `module Spec.Foo.Bar.Baz`

I think I favour option 2, because it's easy to search for when finding filenames, and many editors use the basename of the file in tab names, etc.

Reply:
Ah YES! I favor 2 personally, but I see a lot of usages of 3 and think the team may prefer it.